core/src/main/java/org/apache/calcite/sql2rel/SqlToRelConverter.java

```diff
@@ -4792,6 +4833,32 @@ private void convertValuesImpl(
         true);
   }
 
+  private int[] processColumnsMapping(RelDataType targetRowType, RelOptTable targetTable) {
```

Comment:
I think that it is better to give the code comments to explain what this method does